x/bank/msgs.go

```diff
@@ -206,3 +206,58 @@ func NewOutput(addr sdk.Address, coins sdk.Coins) Output {
 	}
 	return output
 }
+
+//------------------------------
+// IBCSendMsg - IBC transaction of the coin module
+
+// implements sdk.Msg
+type IBCSendMsg struct {
+	DestChain string
+	SendPayload
+}
+
+func (msg IBCSendMsg) Type() string { return "bank" }
+
+func (msg IBCSendMsg) ValidateBasic() sdk.Error {
+	return msg.SendPayload.ValidateBasic()
+}
+
+func (msg IBCSendMsg) Get(key interface{}) interface{} {
+	return nil
+}
+
+func (msg IBCSendMsg) GetSigners() []sdk.Address {
+	return []sdk.Address{msg.SendPayload.SrcAddr}
+}
+
+func (msg IBCSendMsg) GetSignBytes() []byte {
+	b, err := json.Marshal(msg)
```

Comment:
According to #669, `GetSignBytes` function will be gone soon. Until that, it will be more easier to stick with json because of passing `*wire.Codec` to `GetSignBytes`.